erpnext/crm/doctype/appointment/appointment.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+
+from collections import Counter
+from datetime import timedelta
+
+import frappe
+from frappe.model.document import Document
+from frappe.desk.form.assign_to import add as add_assignemnt
+
+
+class Appointment(Document):
+	def validate(self):
+		number_of_appointments_in_same_slot = frappe.db.count('Appointment',filters={'scheduled_time':self.scheduled_time})
```

Comment:
```suggestion
		number_of_appointments_in_same_slot = frappe.db.count('Appointment', filters = {'scheduled_time': self.scheduled_time})
```

Need to fix such styling issues across the code